src/styles/index.scss

```diff
@@ -1,3 +1,244 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+
+  --white: #fff;
+  --blue: #00acdc;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav-item {
+  font-size: 12px;
+  color: #060b35;
+  text-decoration: none;
+  display: block;
+  line-height: 60px;
+
+  &:hover {
+    transform: scale(120%);
+    transition: 300ms linear;
+    color: #00acdc;
+  }
+}
+
+[data-qa='is-active'] {
+  color: #00acdc;
+  position: relative;
+
+  &::after {
+    content: '';
+    display: block;
+    height: 4px;
+    background: #00acdc;
+    width: 100%;
+    border-radius: 8px;
+    position: absolute;
+    bottom: 0;
+  }
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.link {
+  margin-left: 20px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+}
+
+p,
+h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.card-container {
+  --column-count: 1;
+
+  margin: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+
+  gap: 46px;
+
+  row-gap: 48px;
+
+  justify-content: center;
+  justify-content: space-between;
```

Comment:
The 'justify-content' property is declared twice in the '.card-container' class. The second declaration will override the first one. Consider removing the redundant declaration to avoid confusion.